app/apple/server/appleOauthRegisterService.ts

```diff
@@ -0,0 +1,76 @@
+import { KJUR } from 'jsrsasign';
```

Comment:
is there a reason to import `KJUR` instead of `jws` as it was before?

Reply:
its how the package is typed